package.json

```
@@ -20,16 +20,16 @@
         "lint": "eslint index.js"
     },
     "dependencies": {
-        "@sveltejs/adapter-static": "^1.0.0-next.24",
-        "chalk": "^5.0.0"
+        "@sveltejs/adapter-static": "^2.0.2",
+        "chalk": "^5.2.0"
```

I would prefer to keep compatibility with older version, as it's still possible for now.
```suggestion
        "@sveltejs/adapter-static": "^1 || ^2",
        "chalk": "^5.0.0"
```
- `^1 || ^2` will allow all versions 1 and 2 of `@sveltejs/adapter-static`
- `^5.0.0` include version `5.2.0` without limiting the minimum version

(see: https://semver.npmjs.com/)